runtime/src/accounts_db.rs

```diff
@@ -3176,24 +3303,6 @@ impl AccountsDb {
             .fetch_add(recycle_stores_write_elapsed, Ordering::Relaxed);
     }
 
-    fn purge_storage_slots(&self, removed_slots: &HashSet<Slot>) {
```

Comment:
memo: `purge_slot_storage_entries` is the new equivalent fn called in `process_dead_slots`.